api_tests/Register_Jetson_Box_ID.yaml

```diff
@@ -0,0 +1,41 @@
+# Test generated by RoostGPT for test qlue-integration-artillery using AI Type Open AI and AI Model gpt-4
+
+
+config:
+  target: "{{ $processEnvironment.API_HOST }}"
+  plugins:
+    expect: {}
+  payload:
+    path: "Register_Jetson_Box_ID.csv"
+    fields:
+      - "jetson_uuid"
+      - "username"
+      - "clusterId"
+      - "houseId"
+    skipHeader: true
+  phases:
+    - duration: 60
+      arrivalRate: 20
+  environments:
+    dev:
+      target: "https://dev-api-etown.qlue.id/"
+      variables:
+        AUTH_TOKEN: "{{ $processEnvironment.AUTH_TOKEN }}"
+
+scenarios:
+  - name: "Register Jetson Box ID"
+    flow:
+      - post:
+          url: "/RegisterJetsonboxid"
+          headers:
+            Authorization: "Bearer {{ $processEnvironment.AUTH_TOKEN }}"
+          json:
```

Comment:
This api had params not request body, so the query parmas should have been embedded in url